code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -430,11 +430,11 @@ All effects don't start immediately, but rather get worse over time; the rate is
 
 /datum/reagent/consumable/ethanol/goldschlager
 	name = "Goldschlager"
-	description = "100 proof cinnamon schnapps, made for alcoholic teen girls on spring break."
+	description = "100% real cinnamon schnapps, made for alcoholic teen girls on spring break."
```

Comment:
Why this? Proof describes alcohol content.

Reply:
> 
> 
> Why this? Proof describes alcohol content.

Understood, I will fix the changes.

Reply:
I dont think you understood. 100 proof is how americans would write 50% alcohol content.
https://en.wikipedia.org/wiki/Alcohol_proof

Reply:
So different ways of describing the alcohol content, gotcha.

Reply:
Right. And the current text is
```
description = "100% proof cinnamon schnapps, made for alcoholic teen girls on spring break."
```

Reply:
Aye

Reply:
> 
> 
> Right. And the current text is
> 
> ```
> description = "100% proof cinnamon schnapps, made for alcoholic teen girls on spring break."
> ```

Now I understand the issue, I reverted the changes of the description for goldschlager. It's supposed to be 100 proof not 100% proof.